### ISSUE TITLE
datatype conversion issue

### ISSUE THREAD

**Issue by dei79** [Azure/azure-storage-node#40]
Currently I'm facing a critical issue for my application. I expect when I write the following code when creating an entity that it will end up in a property of type double

```javascript
 doubleValue: entGen.Double(0.0),
```

but it results in a string property. If it would be a single issue with 0.0 it would be manageable in my code. But it looks like the same issue happens when using values ending up with .0, e.g.

```javascript
 doubleValue: entGen.Double(85.0),
```
In my application I calculate the data and then put them to a table record. Now after reading them I expect a double but get back a string. 

I digged into this problem and debugged the code a bit. At the end the system generates two different network messages:

```json
{"doubleValue":"0.0"}
```
This ends up in a string. The following one is generated for the value 5.4 works well:

```json
{"doubleValue":5.4}
```
The table store backend generates a double also without type information. I also changed the code in a way that the system generates the oData type indicator but it doesn't help, e.g.:

```json
{"doubleValue":"0.0","doubleValue@odata.type":"Edm.Double"}
```

**Comment by nicklasgav** [comment]
Having the same issue on version 0.5.0. In version 0.5.0 changing `exports.isTypeRequired` in edmhandler.js so that `EdmType.DOUBLE` always returns `true` will solve the problem.

    case EdmType.DOUBLE:
        return true;

Would be great with an update that solves this issue in the next release

**Comment by yaxia** [comment]
ETA of fix is in May release of 2016.

**Comment by yaxia** [comment]
It is fixed in version 1.0. You will get EDM type annotation for a double like "5.0".
